index.html

```diff
@@ -11003,6 +11086,76 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="property" id="aria-textseparation">
+			<pdef>aria-textseparation</pdef>
+			<div class="property-description">
+				<p>Defines how text content of an <a>element</a> is separated from text content of neighboring elements.</p>
```

Comment:
The first 2 sentences of `aria-textseparation` now read:
> Defines how text content of a generic element is separated from text content of adjacent generic elements.
> Specifically, aria-textseparation only applies to text nodes at the boundaries of a generic; it has no effect on separation between text nodes inside the generic, or between other types of elements.

I prefer "adjacent" instead of "neighboring" because neighbor might imply nearby elements, whereas we mean elements that are right beside each other.